section2/reflection.md

```diff
@@ -1,15 +1,23 @@
 ## Section 2 Reflection
 
-1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?  
+  That a closed mindset can be confused for imposter syndrom.  
+  That growth mindset is not always a daily practice but can be a corrective when you feel you are stuck or struggling.  
+2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?  
+I'm going to be honest and say the concept of Growth Mindset doesn't really resonate with me because it seems kind of obvious. Of course there's still more for me to learn
```

Comment:
I see what you mean 😆

I think where we see this manifest in students at Turing a lot is when the going gets tough, they look around and compare themselves and say/think "gosh my classmates are so smart and so much farther ahead of me, I suck" type of stuff, instead of acknowledging all their classmates have done to get there, and the fact that they are capable of doing that too.